### ISSUE TITLE
FNX3-14302 ⁃ [Bug] dynamic bottom toolbar not working as expected (on some websites)

### ISSUE THREAD

**Issue by cadeyrn** [mozilla-mobile/fenix#10978]
### Steps to reproduce
1. open websites and scroll - I saw problems on bugzilla.mozilla.org and github.com, but not (yet) on others.

### Expected behavior
The toolbar hides when you scroll down and appears again when you scroll up.

### Actual behavior
On some websites the toolbar either no longer disappears when you scroll (in most cases on affected websites) or does not appear at all (this happened only once for me on GitHub).

### Device information
* Android device: OnePlus 7T Pro McLaren Edition / Android 10
* Fenix version: master branch revision 248237290e9308b85194319b1416c6d4a7153a06

**Comment by SS1113** [comment]
Seems like a duplicate of #10073, #10045.

**Comment by cadeyrn** [comment]
This is a fairly recent regression, these issues are from April.

**Comment by cadeyrn** [comment]
I didn't notice this before but these issues seem to describe the same so I will close this one.
